.github/workflows/release.yml

```diff
@@ -4,10 +4,11 @@ on:
   push:
```

Comment:
I'm not up to date with github actions, but why do you have 2 files doing the same thing (roughly)